Maybe 6

kosten27 opened the pull request:
Configured liquibase

kosten27 committed:
Added liquibase dependencies and liquibase.properties

kosten27 committed:
Application properties
modify: resources/application.properties: set spring.liquibase and spring datasource properties

kosten27 committed:
Pom xml
modify: pom.xml: deleted liquibase-maven dependencies and plugin, added postgresql dependency

kosten27 committed:
Changelog-master xml
feature: changelog/changelog-master.xml: added changelog-master

kosten27 committed:
Changelog-master xml
modify: changelog/changelog-master.xml: added content

kosten27 committed:
Application properties
modify: resources/application.xml: deleted redundant content

kosten27 committed:
pom
modify: pom.xml: added jpa and hibernate dependencies

kosten27 committed:
changelog-master xml
modify: changelog-master.xml: added changeSets for creating tables: employee, table, product, order, order_item,

kosten27 committed:
changelog-master xml
modify: changelog-master.xml: fixed formatting

Klitsenko committed:
All change set are completed
feature: resources\..\changelog\changelog-master.xml: change sets for tables:
component
component_product
invoice
invoice_item
invoice_type

kosten27 committed:
changelog-master xml
modify: changelog-master.xml: changed table name Table to Customer, change the length of name to 255

Klitsenko committed:
Change columns types
feature: resources\..\changelog\changelog-master.xml: change type from varchar(100 to 255) and to double(15,3)  for tables:
component
component_product
invoice
invoice_item
invoice_type

Klitsenko committed:
Change columns types
feature: resources\..\changelog\changelog-master.xml: change type from varchar(100 to 255) and to double(15,3)  for tables:
component
component_product
invoice
invoice_item
invoice_type

YurasicV commented on src/main/resources/db/changelog/changelog-master.xml:
```diff
@@ -0,0 +1,183 @@
+            <column name="name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="measure_name" type="varchar(255)">
```
I think we must create an integer field "measure_id" instead of "measure_name"

YurasicV commented on src/main/resources/db/changelog/changelog-master.xml:
```diff
@@ -0,0 +1,183 @@
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet id="maybe_6_create_table_invoice_type" author="Dima Klitesnko">
```
Is "invoice_type" a table of database? It seems to me, it must be as Enum

YurasicV commented on src/main/resources/db/changelog/changelog-master.xml:
```diff
@@ -0,0 +1,183 @@
+            <column name="date_created" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="employ_id" type="bigint">
```
Please, correct name of the field to "employee_id"

deivur reviewed (commented):
Please note the @YurasicV comments

Klitsenko committed:
Delete invoice_type change set
modify: resources\..\changelog\changelog-master.xml: delete invoice_type table change data type in column invoice_type_id to bigint.

Klitsenko committed:
Refactor columns change set
modify: resources\..\changelog\changelog-master.xml:
change column measure_name to measure_id and type to bigint;
change column employ_id to employee_id.

kosten27 committed:
changelog-master xml
modify: changelog-master.xml: change wrong field from product to employee in the table Order
